fixtures/assertion-removal/sku.config.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore TS1479
 import type { SkuConfig } from 'sku';
```

Comment:
Apparently this is a thing:
```suggestion
import type { SkuConfig } from 'sku' with { 'resolution-mode': 'import' };
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-3.html#stable-support-resolution-mode-in-import-types

Not ideal if consumers have to do it though, but maybe unavoidable?

Reply:
I'll have a look at it.